src/main/resources/db/migration/V1__Initial_Setup.sql

```diff
@@ -47,7 +47,7 @@ CREATE TABLE m_outbound_messages (
   sms_bridge_id           BIGINT(20)                                      NOT NULL,
   mobile_number           VARCHAR(255)                                    NOT NULL,
   submitted_on_date       TIMESTAMP                                       NOT NULL,
-  delivered_on_date       TIMESTAMP                                       NOT NULL,
+  delivered_on_date       TIMESTAMP                                       NULL DEFAULT NULL,
```

Comment:
@fynmanoj : Don't change existing migration scripts.

Reply:
removed this change